pkg/api/v1/types.go

```diff
@@ -1424,6 +1424,7 @@ type DigitaloceanNodeSpec struct {
 	Size string `json:"size"`
 	// enable backups for the droplet
 	Backups bool `json:"backups"`
+	// DEPRECATED
```

Comment:
maybe also mention why - IPv6 is enabled automatically based on IP Family of the cluster?